Client/Application/AppDelegate.swift

```diff
@@ -294,6 +368,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
                 url = item.value
             case "private":
                 isPrivate = NSString(string: item.value ?? "false").boolValue
+            case "deep-link":
+                guard let value = item.value else { break }
+                Router.shared.routeURL(value)
```

Comment:
If the assert passes, then it's fine--this code shouldn't execute if we're not on the main thread anyway.